src/TransitFunctionAppTest/ModelsTests/KioskDeviceTest.cs

```diff
@@ -102,6 +102,37 @@ public void TestKioskPurchaseTicket()
             Assert.AreEqual(methodresult.Status, 200); // got back an ok
         }
 
+        [Test]
+        public void TestPurchaseDenied()
+        {
+            FakeDeviceClient fakeDeviceClient = new FakeDeviceClient();
+            FakeEventScheduler fakeScheduler = new FakeEventScheduler();
+
+            TestContext.WriteLine(">> Testing the Kiosk Device's Low Stock notification..");
+
+            PurchaseTicketPayload approvePurchaseMethodkRequest = new PurchaseTicketPayload()
+            {
+                DeviceId = deviceconfig.DeviceId,
+                DeviceType = deviceconfig.DeviceType,
+                MessageType = "Purchase",
+                IsApproved = false
+            };
+
+            // create our test device
+            KioskDevice device = new KioskDevice(deviceconfig, fakeDeviceClient, fakeScheduler);
+
+            TestContext.WriteLine("/n>> Purchasing tickets, shouldn't throw event");
```

Comment:
did you mean "\n" ? If that's the case, please make an empty WriteLine instead

Reply:
yes, its there's to make the test log more readable. Are you sure we need an entirely new line of code just for readability?

Reply:
I highly prefer splitting them for the following reasons:
* It's more readable in the Code, more explicit, and matches the output better.
* It's more standard, in Windows we'll need the carriage return too.
Anyway it's up to you, but at the very least I believe you need to fix the /n to \n.

Reply:
kk

Reply:
changed